app/javascript/packages/document-capture/components/acuant-capture.scss

```diff
@@ -0,0 +1,37 @@
+// Note: ID specifier necessary only until USWDS upgraded.
+#document-capture-form .document-capture-acuant-capture { // scss-lint:disable IdSelector
+  max-width: 375px;
+
+  .usa-file-input:not(.usa-file-input--has-value) {
+    @include u-margin-top(1);
+    position: relative;
+
+    // Note: Lint disable is no longer necessary after above ID specifier is removed.
+    .usa-file-input__target { // scss-lint:disable SelectorDepth
+      align-items: center;
+      bottom: 0;
+      display: flex;
+      flex-direction: column;
+      height: 100%;
+      justify-content: center;
+      left: 0;
+      margin-top: 0;
+      position: absolute;
+      right: 0;
+      top: 0;
+    }
+
+    &::after {
+      content: '';
+      display: block;
+      // 2.125" x 3.375" are common standard ID dimensions
+      padding-bottom: ((2.125 / 3.375) * 100) + unquote('%');
```

Comment:
I know this just got copy-pasted from the other file, but since we have the same comment and line twice, is it worth taking advantage of SASS/SCSS and making a variable for this? `$common-id-card-ratio` or something

Reply:
Yeah, makes sense.

Could also use a placeholder or mixin to apply the padding (and maybe `display: block`), since unless we're willing to assign the ratio as the percentage value, we'd still have to do some math + concatenation:

```scss
padding-bottom: ($common-id-card-ratio * 100) + unquote('%');
```

Reply:
Updated to use SASS placeholder in 2df9969.